### ISSUE TITLE
What are we supposed to do with the raw volume exports?

### ISSUE THREAD

**Issue by Crackheadlegend** [NVlabs/instant-ngp#1279]
Is there any way to import these bin files into any other program or to convert them into point clouds?
Are the just a useless file at the moment?

**Comment by arnabdeypolimi** [comment]
Yes, you can use a small python script to convert sigmas to point cloud. the shape of the bin file is 32X32X32X4 (RGBA) where A is the sigma. Basically, nerf radiance and sigma for XYZ point.

**Comment by mojoyup** [comment]
Where do they save to?
